Sources/StreamChat/Query/MessageSearchQuery.swift

```diff
@@ -86,7 +109,7 @@ public struct MessageSearchQuery: Encodable {
     
     public let sort: [Sorting<MessageSearchSortingKey>]
     
-    public var pagination: Pagination?
+    public var pagination: SearchPagination?
```

Comment:
Isn't this a breaking change? 🤔 Can't we add a new property `cursor` to `Pagination` to avoid the breaking change? And then `Pagination` would only have 2 constructors, one where we use the cursor, and one where use Offset and limit.

Reply:
Good catch!